app/controllers/provider_foreman_controller.rb

```diff
@@ -455,19 +491,26 @@ def features
      {:role     => "configured_systems_filter_accord",
       :role_any => true,
       :name     => :cs_filter,
-      :title    => _("Configured Systems")}
-    ].map do |hsh|
+      :title    => _("Configured Systems")},
+     {:role     => "cm_job_templates_accord",
+      :role_any => true,
+      :name     => :cm_job_templates,
+      :title    => _("Ansible Job Templates")}
+     ].map do |hsh|
       ApplicationController::Feature.new_with_hash(hsh)
     end
   end
 
   def build_configuration_manager_tree(type, name)
     @sb[:open_tree_nodes] ||= []
 
-    tree = if name == :configuration_manager_providers_tree
+    tree = case name
+           when :configuration_manager_providers_tree
              TreeBuilderConfigurationManager.new(name, type, @sb)
-           else
+           when :cs_filter_tree
```

Comment:
Was `:cs_filter_tree` the only other value of name in the past?  It was much more open ended.  (don't know if that was intentional)